CI fails on Windows tests

CI fails on Windows tests on the step `Run goanpeca/setup-miniconda@v1.6.0` with the next message:
```
CondaError: Cannot link a source that does not exist. C:\Miniconda\Scripts\conda.exe
Running `conda clean --packages` may resolve your problem.

##[error]The process 'C:\Miniconda\condabin\conda.bat' failed with exit code 1
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -299,6 +299,7 @@ jobs:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
       - name: conda clean
+        uses: goanpeca/setup-miniconda@v1.6.0
         shell: bash -l {0}
         run: conda clean --packages
       - uses: goanpeca/setup-miniconda@v1.6.0

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -298,6 +298,9 @@ jobs:
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
+      - name: conda clean
+        shell: bash -l {0}
+        run: conda clean --packages
       - uses: goanpeca/setup-miniconda@v1.6.0
         with:
           activate-environment: modin

--- a/.github/workflows/push.yml
+++ b/.github/workflows/push.yml
@@ -124,6 +124,9 @@ jobs:
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
+      - name: conda clean
+        shell: bash -l {0}
+        run: conda clean --packages
       - uses: goanpeca/setup-miniconda@v1.6.0
         with:
           activate-environment: modin

--- a/.github/workflows/push.yml
+++ b/.github/workflows/push.yml
@@ -125,6 +125,7 @@ jobs:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
       - name: conda clean
+        uses: goanpeca/setup-miniconda@v1.6.0
         shell: bash -l {0}
         run: conda clean --packages
       - uses: goanpeca/setup-miniconda@v1.6.0

